app/components/publications/publication/translations/translation-upload-modal.hbs

```diff
@@ -0,0 +1,94 @@
+<Auk::Modal>
+  <Auk::Modal::Header
+    @resizeable={{false}}
+    @title={{t
+      "upload-translation-document"
+      identifier=@translationSubcase.publicationFlow.identification.idName
+    }}
+    @onClose={{perform this.cancelTranslation}}
+  />
+  <Auk::Modal::Body>
+    <div class="auk-u-p-4">
+      <Auk::FileUploader
+              @fileQueueName={{this.fileQueueName}}
+              @fullHeight={{eq this.newPieces.length 0}}
+              @multiple={{true}}
```

Comment:
Multiple file upload lijkt niet te werken. Er worden wel meerdere files opgeladen, maar daarna kan je maar van 1 file de details bewerken. Volgens mij is het OK als we momenteel maar 1 file tegelijk toelaten. Vraagt misschien nog wat opkuiswerk van de overbodige `this.newPieces` dan.